### ISSUE TITLE
create traffic light training set with simulator data

### ISSUE THREAD

**Issue by alanswx** [evenator/CarND-Capstone#11]
we downloaded a lot of udacity and LISA data, we need to convert it into a different format to train the tensorflow models. If we get it working, we will then post the format we want the data in from the simulator

**Comment by alanswx** [comment]
probably TF.records format

**Comment by alanswx** [comment]
each image with a  CSV or space that includes:

filename xmin ymin xmax ymax occluded label attr
1478019966187511711.jpg 926 506 948 540 0 "trafficLight" "Red"
1478019966187511711.jpg 926 506 948 540 0 "trafficLight" "Green"

**Comment by evenator** [comment]
Can you post a link to the data you need labeled?

**Comment by evenator** [comment]
Or do you need me to capture data from the simulator?

**Comment by alanswx** [comment]
It needs to be captured from the simulator.  I wonder if it can be captured with the label as well. I am going to work on some training scripts first, if you don't get to it I will try to generate the data tomorrow.

**Comment by evenator** [comment]
No bounding boxes yet, but I have the beginnings of an image set. https://drive.google.com/open?id=0B8ZBMXqFm4LjNTZRajhiakZ0eHM

**Comment by alanswx** [comment]
This is almost done. I have a mediocre version of the classifier trained after doing about 100 bounding boxes. I am going to try to finish up 1000 bounding boxes in the next couple of days. It is not a blocker anymore.
